updater/fetchers/alpine/alpine.go

```diff
@@ -107,13 +143,12 @@ func parseVulnsFromNamespace(repositoryPath, namespace string) (vulns []database
 	}
 	defer file.Close()
 
-	switch namespace {
-	case "v3.3":
-		vulns, err = parse33YAML(file)
-	case "v3.4":
-		vulns, err = parse34YAML(file)
+	parseFunc, exists := parsers[namespace]
+	if !exists {
+		return
```

Comment:
All these are much better. Please just add a Warning here as well as an updater Note for that so we can be alerted when we need to take action. See https://github.com/coreos/clair/blob/master/updater/fetchers/ubuntu/ubuntu.go#L139